.github/workflows/integration.yaml

```diff
@@ -167,3 +167,43 @@ jobs:
             unicode \
             unicode/utf16 \
             unicode/utf8
+
+
+  wasi-testsuite:
```

Comment:
You have to install the python dependencies via `pip install -r requirements/dev.txt` and please configure the proper caching for them`

https://github.com/WebAssembly/wasi-testsuite/blob/e5bcece941f66f8f6c845a9add32f9f99e444a2c/.github/workflows/test-runner.yml#L31

Reply:
should be ok now